watertap/unit_models/zero_order/tests/test_clarifier_zo.py

```diff
@@ -0,0 +1,299 @@
+
+    @pytest.mark.component
+    def test_load_parameters(self, model):
+        data = model.db.get_unit_operation_parameters("clarifier")
+
+        model.fs.unit.load_parameters_from_database(use_default_removal=True)
+
+        assert model.fs.unit.recovery_frac_mass_H2O[0].fixed
+        assert model.fs.unit.recovery_frac_mass_H2O[0].value == \
+            data["recovery_frac_mass_H2O"]["value"]
+
+        for (t, j), v in model.fs.unit.removal_frac_mass_solute.items():
+            assert v.fixed
+            if j == "foo":
+                assert v.value == data["default_removal_frac_mass_solute"]["value"]
+            else:
+                assert v.value == data["removal_frac_mass_solute"][j]["value"]
+
+        assert model.fs.unit.energy_electric_flow_vol_inlet.fixed
+        assert model.fs.unit.energy_electric_flow_vol_inlet.value == data[
+            "energy_electric_flow_vol_inlet"]["value"]
+
+    @pytest.mark.component
+    def test_degrees_of_freedom(self, model):
+        assert degrees_of_freedom(model.fs.unit) == 0
+
+    @pytest.mark.component
+    def test_unit_consistency(self, model):
+        assert_units_consistent(model.fs.unit)
+
+    @pytest.mark.component
+    def test_initialize(self, model):
+        initialization_tester(model)
+
+    @pytest.mark.solver
+    @pytest.mark.skipif(solver is None, reason="Solver not available")
+    @pytest.mark.component
+    def test_solve(self, model):
+        results = solver.solve(model)
+
+        # Check for optimal solution
+        assert_optimal_termination(results)
+
+    @pytest.mark.solver
+    @pytest.mark.skipif(solver is None, reason="Solver not available")
+    @pytest.mark.component
+    def test_solution(self, model):
+        assert (pytest.approx(1.4e-2, rel=1e-5) ==
+                value(model.fs.unit.properties_in[0].flow_vol))
+        assert (pytest.approx(214.2857, rel=1e-5) ==
+                value(model.fs.unit.properties_in[0].conc_mass_comp["tss"]))
+        assert (pytest.approx(71.4286, rel=1e-5) ==
+                value(model.fs.unit.properties_in[0].conc_mass_comp["foo"]))
+        assert (pytest.approx(1.10175e-2, rel=1e-5) ==
+                value(model.fs.unit.properties_treated[0].flow_vol))
+        assert (pytest.approx(2.4980, rel=1e-5) ==
+                value(model.fs.unit.properties_treated[0].conc_mass_comp["tss"]))
+        assert (pytest.approx(90.7645, rel=1e-5) ==
+                value(model.fs.unit.properties_treated[0].conc_mass_comp["foo"]))
+        assert (pytest.approx(2.9825e-3, rel=1e-5) ==
+                value(model.fs.unit.properties_byproduct[0].flow_vol))
+        assert (pytest.approx(996.648, rel=1e-5) ==
+                value(model.fs.unit.properties_byproduct[0].conc_mass_comp["tss"]))
+        assert (pytest.approx(0, rel=1e-5) ==
+                value(model.fs.unit.properties_byproduct[0].conc_mass_comp["foo"]))
+        assert (pytest.approx(0, rel=1e-5) ==
```

Comment:
It appears that the electricity consumption/intensity is set to zero here. Is this something we expect to be universal, or is it just that we don't have sufficient data? If the former, then I have two questions/comments:

1. Should we even include electricity consumption in this model? It might be worth it for consistency, but I am not sure.
2. If we are going to include electricity consumption, should we have a new "zero_consumption` method that just fixes the consumption to zero to avoid having the additional constraints?

Reply:
I think that there might be energy consumption related to pumping, but the yaml file doesn't include any estimates for that. I think we could just leave this as zero for now and either (a) revise to a non-zero value later as info becomes available or (b) leave room for the user to provide their own value for electricity intensity.